apis/beacon/states/pending_deposits.yaml

```diff
@@ -0,0 +1,51 @@
+get:
+  operationId: "getPendingDeposits"
+  summary: "Get State Pending Deposits"
+  description: |
+    Returns pending deposits for state with given 'stateId'. Should return 400 if requested before electra.
+  tags:
+    - Beacon
+  parameters:
+    - name: state_id
+      in: path
+      $ref: '../../../beacon-node-oapi.yaml#/components/parameters/StateId'
+  responses:
+    "200":
+      description: Success
+      content:
+        application/json:
+          schema:
+            title: GetPendingDepositsResponse
+            type: object
+            required: [execution_optimistic, finalized, data]
```

Comment:
do we also wanna add version meta? it's seems unlikely that the type will change in future hard forks but as a general rule I think it would be good to have it if the api uses a container of the CL spec as return type

Reply:
i may have just copied the wrong one, i think yes we should add the version and headers...